### ISSUE TITLE
Addon breaks menus on the secure desktop

### ISSUE THREAD

**Issue by tspivey** [marlon-sousa/EnhancedDictionaries#4]
STR:
1. Switch to NVDA on the secure desktop, with this addon installed.
2. Activate one of the menu items, like settings.

I get this traceback:
```
IO - speech.speech.speak (07:34:05.633) - MainThread (20660):
Speaking ['Settings...', 's', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (07:34:06.184) - winInputHook (22228):
Input: kb(laptop):enter
DEBUG - external:globalPlugins.EnhancedDictionaries.ConfigManager.getActiveProfile (07:34:06.190) - MainThread (20660):
safe profile
ERROR - unhandled exception (07:34:06.191) - MainThread (20660):
Traceback (most recent call last):
  File "C:\Users\tyler\AppData\Roaming\nvda\addons\EnhancedDictionaries\globalPlugins\EnhancedDictionaries\__init__.py", line 76, in onVoiceDictionaryCommand
    dic = dictHelper.getDictionary("voice")
  File "C:\Users\tyler\AppData\Roaming\nvda\addons\EnhancedDictionaries\globalPlugins\EnhancedDictionaries\dictHelper.py", line 71, in getDictionary
    return _getVoiceDictionary(profile)
  File "C:\Users\tyler\AppData\Roaming\nvda\addons\EnhancedDictionaries\globalPlugins\EnhancedDictionaries\dictHelper.py", line 50, in _getVoiceDictionary
    if not profile.name or _hasVoiceDictionaryProfile(profile.name, synth.name, dictionaryFilename):
AttributeError: 'NoneType' object has no attribute 'name'
```

An easier way to debug this is to restart NVDA with `--secure --debug` on the normal desktop, cause the error, restart NVDA and look at `%temp%\nvda-old.log`.

**Comment by marlon-sousa** [comment]
Hello,
Thanks for reporting. I couldn’t replicate it. What I did?

I pressed Ctrl plus alt plus delete to access secure screen. Then, I tried to access several menus and didn’t get any error. what is your version of windows and also the versions of the addom and NVDA?

anyways, the trace back is interesting enough that I can start working on it. I should have a new version in a couple of days. Obrigado,
Marlon
